frontend/app/src/people/widgetViews/OrganizationDetails.tsx

```diff
@@ -73,6 +72,8 @@ const OrganizationDetails = (props: { close: () => void; org: Organization | und
 
   const isOrganizationAdmin = props.org?.owner_pubkey === ui.meInfo?.owner_pubkey;
 
+  const editOrgDisabled =
+    !isOrganizationAdmin && !userHasRole(main.bountyRoles, userRoles, 'ADD USER');
```

Comment:
this is the same thing as addUserDisabled we might want to just rename that variable to make more sense instead of redefining the same thing twice